### ISSUE TITLE
Add golog switch

### ISSUE THREAD

**Issue by roidelapluie** [mlimaloureiro/golog#22]
golog switch task should:

- stop current task if any
- start a new task

It should error if there is more than one started task
It should not error if more no task is started

**Comment by mlimaloureiro** [comment]
makes sense @roidelapluie I'll have a look into it. I was with no time to follow up on golog, but I'll dedicate some time again to it.

Thank you for your suggestion
